conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,14 @@
+# --- !Ups
+
+ALTER TABLE `users`
+ADD COLUMN `date_created` DATETIME NULL AFTER `last_login`;
+ADD COLUMN `created_by` DATETIME NOT NULL AFTER `date_created`;
```

Comment:
Change:

ADD COLUMN `created_by` DATETIME NOT NULL AFTER `date_created`;

To:

ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;

Then we can re-test, but looks good otherwise.